sqldb/rust/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "wasmcloud-interface-sqldb"
+version = "0.1.0"
+description = "interface for actors to issue http/https requests (wasmcloud:httpclient)"
```

Comment:
Suuuuper nitpicky, but fields like this that are published on crates should start with an uppercase letter.